index.html

```diff
@@ -7,28 +7,65 @@
     <title>Awesome Books</title>
```

Comment:
Amazing job building this webpage just fix this minor change, when the book list is open have an active class on the navbar so the user can know on which page he is currently on, according to the wireframes provided.

![Screenshot from 2023-08-10 14-52-53](https://github.com/memicelis/awesome-books/assets/118063058/d21d12a9-fb94-4e98-8762-5a0c03182aa6)